aoc/src/main/scala/aoc2015/Day01.scala

```diff
@@ -0,0 +1,42 @@
+object Day01 extends App:
+  import scala.io.Source
+  import scala.collection.mutable.ListBuffer
+
+  val lines: String =
+    Source
+      .fromFile("aoc/src/resources/aoc2015/Day01")
+      .getLines
+      .mkString
+
+  def findfloor(current: Int, instructions: String): Int = 
+    if (instructions.isEmpty) {
```

Comment:
more idiomatic is if-then syntax with the advantage that curly brackets are not needed, ie.

```
if instructions.isEmpty then
  findfloor(current + 1, instructions.tail)
else if instructions.head.toString == ")" then
  findfloor(current - 1, instructions.tail)
else
  0
```